test/runtests.jl

```diff
@@ -1,6 +1,10 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 using Test, LinearAlgebra, SparseArrays
 
+if Base.get_bool_env("SPARSEARRAYS_AQUA_TEST", false)
```

Comment:
Might be easier to just use
```suggestion
if get(ENV, "SPARSEARRAYS_AQUA_TEST", nothing) === "true"
```
since this isn't something people will configure in general (and thus not forget whether it is 1 or true).